src/mixer/state.ts

```diff
@@ -53,6 +53,7 @@ interface MicState {
   volume: 1 | 0;
   baseGain: number;
   id: string | null;
+  processing: boolean;
```

Comment:
Would appreciate a more descriptive name (not too fussed though)

Reply:
Hmm, I did start with something else, but I think it's pretty obvious given it's a boolean.